src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -24,12 +27,20 @@
 public class MainWindow extends UiPart<Stage> {
 
     private static final String FXML = "MainWindow.fxml";
+    private static final String DARK_THEME = "dark";
+    private static final String LIGHT_THEME = "light";
+    private static final String DARK_THEME_PATH = "/view/DarkTheme.css";
+    private static final String LIGHT_THEME_PATH = "/view/LightTheme.css";
 
     private final Logger logger = LogsCenter.getLogger(getClass());
 
     private Stage primaryStage;
     private Logic logic;
 
+    //private String theme = "light";
+    private String theme;
+    private Scene scene;
```

Comment:
Is there any significance behind this comment? Apologies, but I am having trouble understanding this.

Reply:
oops, missed out on removing this